web/src/hooks/queries/useValidation.ts

```diff
@@ -35,14 +43,11 @@ const runValidation = async (pep: FileList | string | undefined, schema: string
   return result;
 };
 
-export const useValidation = (
-  pep: FileList | string | undefined,
-  schema: string | undefined,
-  enabled: boolean = false,
-) => {
-  return useQuery(['validation', pep, schema], () => runValidation(pep, schema), {
-    enabled: enabled && pep !== undefined && schema !== undefined && pep.length > 0 && schema?.length > 0,
+export const useValidation = (params: ValidationParams) => {
+  return useQuery(['validation', params.pep, params.schema, params.schema_registry], () => runValidation(params), {
```

Comment:
~~This looks like it might need `prettier` run on it.~~

Nvm, my browser window was just too narrow.